src/components/hovedmal.tsx

```diff
@@ -0,0 +1,44 @@
+import { OrNothing, StringOrNothing } from '../utils/felles-typer.ts';
+import { HovedmalType } from '../data/api/datatyper/kodeverk14aData.ts';
+import { Hovedmal } from '../data/api/datatyper/siste14aVedtak.ts';
+import EMDASH from '../utils/emdash.ts';
+import { Errormelding, Laster } from './felles/minikomponenter.tsx';
+import { useKodeverk14a } from '../data/api/fetch.ts';
+import { DobbeltInformasjon } from './felles/dobbelinfo.tsx';
+import { formaterDato } from '../utils/formater.ts';
+
+interface Props {
```

Comment:
Kan vi rename denne til HovedmalProps?

Reply:
Vi kan det, men da syns jeg det bør gjøres overalt. Det er 13 steder i koden der props er kalt bare Props.

Reply:
Den er ikke så illa for meg om vi beholder Props foreløpig. :)